src/finn/custom_op/fpgadataflow/globalaccpool_batch.py

```diff
@@ -75,16 +75,19 @@ def get_folded_input_shape(self):
     def get_normal_output_shape(self):
         ch = self.get_nodeattr("NumChannels")
         vecs = list(self.get_nodeattr("numInputVectors"))
-        oshape = tuple([vecs[0]] + [ch])
+        if len(vecs) == 1:
+            oshape = tuple(vecs + [ch])
```

Comment:
is this a valid use case? in which cases(s) does this op get an op that looks like (1, ifm_ch)?